Summary: From: Curt Brune <curt@brune.net>

implicit-fallthrough.patch

When compiling using gcc-7 with -Werror=implicit-fallthrough enabled,
implicit case statement fallthroughs are not permitted.

This patch uses the gcc attribute "fallthrough" to make the
fallthrough explicit.

diff --git a/src/authvar.c b/src/authvar.c
index ad659ca..0c5d752 100644
--- a/src/authvar.c
+++ b/src/authvar.c
@@ -511,6 +511,7 @@ main(int argc, char *argv[])
 	case IMPORT|SET:
 	case IMPORT|SIGN|SET:
 		fprintf(stderr, "authvar: not implemented\n");
+		__attribute__ ((fallthrough));
 	case IMPORT|SIGN|EXPORT:
 	default:
 		fprintf(stderr, "authvar: invalid flags: ");
